molecule/core.py

```diff
@@ -215,7 +215,7 @@ def _create_inventory_file(self):
         inventory = ''
         # TODO: for Ansiblev2, the following line must have s/ssh_//
         host_template = \
-            '{} ansible_ssh_host={} ansible_ssh_port={} ansible_ssh_private_key_file={} ansible_ssh_user={}\n'
+            '{} ansible_ssh_host={} ansible_ssh_port={} ansible_ssh_private_key_file={} ansible_ssh_user={} ansible_ssh_extra_args="-o StrictHostKeyChecking=no -o UserKnownHostsFile=/dev/null"\n'
```

Comment:
Thanks for this! Only thing I'd ask is to follow the existing style of having these values passed into the .format() below and wrapping at 80 chars. Running it through yapf should do the code formatting for you.